paasta_tools/native_mesos_scheduler.py

```diff
@@ -37,6 +38,7 @@ def main(argv):
             service_name=service,
             instance_name=instance,
             cluster=cluster,
+            staging_timeout=args.staging_timeout,
```

Comment:
here we set the type in the argparser of ``native_mesos_scheduler``. we didn't do it for the ``remote_run`` parser - I should add it there though so I can remove the float cast.